docker-compose.yml

```diff
@@ -1,19 +1,62 @@
-version: '3'
+version: '2.1'
 services:
-  api:
-    image: openrecord/turntable:latest
+  vinyl:
+    build:
+      context: .
+      dockerfile: docker/vinyl/Dockerfile
     ports:
-      - 7001:7001
+      - 8080:8080
     environment:
-      PORT:     # Optionally specify a port for the API here.
-      DB_HOST: mysql
-    depends_on:
-      - mysql
-  mysql:
-    image: mysql:5.7
-    volumes:
-      - ./docker/mysql/mount:/var/lib/mysql # Add a mount for all mysql files, so you can persist across container tear-downs.
+      HOST: docker
+    networks: [main]
+
+  #TODO: wait on prisma healthcheck
+  turntable-deploy:
+    image: openrecord/turntable:prisma-cli
+    depends_on: [prisma]
+    command: npx prisma deploy
     environment:
-      MYSQL_ROOT_PASSWORD: password
+      ENDPOINT: http://prisma:4466
+    networks: [main]
+
+  #TODO: Add healthcheck
+  prisma:
+    image: prismagraphql/prisma:1.15
+    restart: always
+    environment:
+      PRISMA_CONFIG: |
+        port: 4466
+        databases:
+          default:
+            connector: postgres
+            host: postgres
+            port: 5432
+            user: prisma
+            password: prisma
+            migrations: true
```

Comment:
We should very quickly find a way to avoid copying the config options from turntable in here, maybe we can just use the `openrecord/turntable` image? We'll quickly have to find a convention for tagging images so we can test backend changes locally.

Reply:
Yeah. When we make a backend we'll wrap it all into an openrecord/turntable image which is self-contained and simple.